pkg/utils/pathing/helpers.go

```diff
@@ -0,0 +1,28 @@
+package pathing
+
+import (
+	"runtime"
+	"strings"
+)
+
+// returns a string (filepath) with appropriate slashes, dependent on OS. Optional parameters and method overloading unavailable in Go.
+func SanitizeFilepath(filepath string) string {
+	os := runtime.GOOS
+	switch os {
+	case "windows":
+		return strings.Replace(filepath, "\\", "/", -1)
+	default:
+		return filepath
+	}
+}
+
+// returns a string (filepath) with appropriate slashes, dependent on OS if error is empty. Optional parameters and method overloading unavailable in Go.
+func SanitizeFilePathWithError(filepath string, err error) (string, error) {
```

Comment:
What would this function be used for out of curiosity?

Reply:
There were some places where a function is called that returns a path and an error. So for those cases this could be used.